DPB_Converger_config.ini

```diff
@@ -0,0 +1,29 @@
+[SOFTWARE]
+# choose software, either ThunderSTORM or rapidSTORM
+software = ThunderSTORM
+
+[INPUT_DIRS]
+# list each coverslip
+dir01 = D:\exampledir\CS1
+
+
+[PARAMETERS_GLOBAL]
+# define initial global swift parameters for the swift.bat. tau in ms, displacements and diffraction limit in nm, probabilities as numbers (not percentage)
+tau = 20
+exp_displacement = 120
+max_displacement = 2.5
+max_displacement_pp = 3.5
+p_bleach = 0.05
+p_switch = 0.01
+diffraction_limit = 42
+
+[P_BLEACH_PARAMETERS]
+#set parameter for the pBleach fit
```

Comment:
expand the comment and define the unit of camera intengration time + briefly explain what initial k and number of points are.